Scripts/script-ParseEmailHeaders.yml

```diff
@@ -0,0 +1,38 @@
+commonfields:
+  id: ParseEmailHeaders
+  version: -1
+name: ParseEmailHeaders
+script: |-
+  from email.parser import HeaderParser
+
+  ec = {"Email.Headers" : {}}
+  md = ""
+
+  # Get the file path of the email file as it lokated in Demisto server
+  filePath = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryid']})[0]['Contents']['path']
```

Comment:
Did you check all scenarios? What if file does not exist? what if it is not an email file?

Reply:
Fixed